README.md

```diff
@@ -1,57 +1,250 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Math Magicians
+<!-- TABLE OF CONTENTS -->
 
-# Description of the project.
-- To Set up the environment and tools needed to develop a React application.
-- Configure with special linters. 
+# 📗 Table of Contents
 
-# Built With
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-- React
-- Webpack and Babel
+<!-- PROJECT DESCRIPTION -->
 
-# Key Features
+# 📖 [Math Magicians] <a name="about-project"></a>
 
-- Make simple calculations.
-- Read a random math-related quote.
+> Describe your project in 1 or 2 sentences.
 
-# Future Features
+**Math Magician** is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
 
-- To add 3 pages to the website which will display different information
+Make simple calculations.
+Read a random math-related quote.
 
-# Getting Started
+## 🛠 Built With <a name="built-with">HTML, CSS, JS</a>
 
-**To get a local copy up and running follow these simple steps.**
+### Tech Stack <a name="tech-stack">React</a>
 
-Run the command below from the terminal
+> Describe the tech stack and include only the relevant sections that apply to your project.
 
-- git clone (https://github.com/teke85/Math-Magician.git) in your terminal
-- cd Math-Magician
-- run: npm start
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
-## Demo
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
 
-Not Available
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```

Comment:
- [ ] [OPTIONAL] Since your project does not have a Server and Database, please edit your README and remove the highlighted part 👍